src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FootnoteServiceTests.cs

```diff
@@ -210,450 +198,304 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
         [Fact]
         public async Task CreateFootnote_MultipleFootnotesHaveExpectedOrder()
         {
-            var subject = new Subject();
+            var release = _releaseGenerator.Generate();
+            var subject = _subjectsGenerator.Generate();
 
-            // Create a release which already has some existing footnotes
-            var release = new Release
-            {
-                Footnotes = new List<ReleaseFootnote>
-                {
-                    new()
-                    {
-                        Footnote = new Footnote
-                        {
-                            Content = "Footnote 1",
-                            Order = 0,
-                            Subjects = new List<SubjectFootnote>
-                            {
-                                new()
-                                {
-                                    Subject = subject
-                                }
-                            }
-                        }
-                    },
-                    new()
-                    {
-                        Footnote = new Footnote
-                        {
-                            Content = "Footnote 2",
-                            Order = 1,
-                            Subjects = new List<SubjectFootnote>
-                            {
-                                new()
-                                {
-                                    Subject = subject
-                                }
-                            }
-                        }
-                    }
-                }
-            };
-
-            var releaseSubject = new ReleaseSubject
-            {
-                Release = release,
-                Subject = subject
-            };
+            var footnotes = _footnoteGenerator
+                .WithSubjects(ListOf(subject))
+                .GenerateList(2);
 
-            var contextId = Guid.NewGuid().ToString();
+            var releaseSubject = _releaseSubjectGenerator
+                .ForInstance(s => s.Set(rs => rs.Release, release))
+                .ForInstance(s => s.Set(rs => rs.Subject, subject))
+                .Generate();
```

Comment:
I was surprised by this because I figured that you'd probably not want an extension method for every possible property that could be set, and instead implement them for the more complex cases only (e.g. where back-references need to be set or some intermediary link table items are actually being generated.  Otherwise the generator extensions get pretty bloated with setters for loads of properties.  Don't get me wrong though, I'm personally happy to do it for readability, but I figured you'd not want it that way?

Reply:
Worked through most of these now - I'm happy to do it this way as it's much more readable.

Reply:
Done ta